keyboards/ydkb/mountain/config.h

```diff
@@ -0,0 +1,56 @@
+/* Copyright 2020
```

Comment:
This should be handled as a separate PR, since it's for a full keyboard, not just a keymap. 

Please pull out these files, and add them to a separate PR.